### ISSUE TITLE
Two related ERRORS from WCAG 2.0 that should be fixed (Notes on 1.3.3 and 1.4.1)

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag#267]
[WCAG 2.1 Comment 280](https://github.com/w3c/wcag21/issues/280) is a WCAG 2.0 comment, so filing this issue here.

**Comment by awkawk** [comment]
The note in 1.3.3 isn't incorrect, but I agree that it is confused by the lack of color in the list of visual properties in SC 1.3.3 and further by the lack of a failure that addresses this.

Perhaps we should create a failure with a "click the red button to proceed to the next step" instruction and a red button with the text "next step"?

The note on 1.4.1 is genuinely confusing.
current: 
Note: This success criterion addresses color perception specifically. Other forms of perception are covered in Guideline 1.3 including programmatic access to color and other visual presentation coding.

I agree that removing this note will reduce confusion, especially if there was a good failure of 1.3.1 that highlighted how color conveys relationships that need to be expressed.

**Comment by awkawk** [comment]
This would be the fix - 
https://github.com/w3c/wcag21/pull/295 (this is in the WCAG 2.1 repo)

**Comment by awkawk** [comment]
This was addressed and is integrated into the WCAG 2.0 errata: https://www.w3.org/WAI/WCAG20/errata/
